Crypt/Mechanisms/CryptMechanism.swift

```diff
@@ -161,10 +161,10 @@ class CryptMechanism: NSObject {
       guard let data = getHintData(key: needsEncryptionHintKey) else {
         return false
       }
-      guard let value = NSKeyedUnarchiver.unarchiveObject(with: data as Data) else {
+      guard let value = try! NSKeyedUnarchiver.unarchivedObject(ofClass: NSNumber.self, from: data as Data) else {
```

Comment:
Probably should follow the same approach here that @mactroll showed above; rather than disabling error propogation, use the `try?` approach to handle the error with an optional. Then the guard's else will handle it, and there won't be a runtime error.

Reply:
updated, Thanks!